src/picongpu/include/particles/ionization/ionizationMethods.hpp

```diff
@@ -63,7 +63,7 @@ namespace ionization
 
             /* each thread sets the multiMask hard on "particle" (=1) */
             childElectron[multiMask_] = 1;
-            const uint32_t weighting = parentIon[weighting_];
```

Comment:
fun fact: in this routine, one could skip reading `weighting` all together since only a `getMass` ratio is calculated.